assets/components/atoms/event-teaser/event-teaser-highlighted.twig

```diff
@@ -0,0 +1,24 @@
+<a href="#" class="event-teaser event-teaser-highlighted card p-4">
+
+  <picture class="card-img-top">
+    <img src="./images/event_teaser_large.jpg" class="d-block" title="professor Xile Hu" aria-labelledby="event-teaser-label" />
```

Comment:
Use `alt` attribute instead of re-labelling with `aria-labelledby`

Reply:
I used aria-labelledby based on the order of precedence found on https://a11yproject.com/posts/title-attributes/ , but can totally be an alt instead :) what do you think?

Reply:
Ever use `alt` before anything else, it's a matter of native a11y and proper image SEO. It seems to be a good idea to use `aria-labelledby`, but using the simpler and the most default markup is always a better choice. As Naomi Watson said during her talk, don't try to fix HTML until you really break it on purpose. ARAI are most of the time here to fix something you break or that is not clear HTML understandable.